### ISSUE TITLE
When using the package in Flutter 3.10, an error appears

### ISSUE THREAD

**Issue by StarProxima** [MadBrains/App-Runner-Flutter#12]
Example: 

```dart
void main() async {
  final widgetConfiguration = WidgetConfiguration(
    child: AppBuilder<void>(
      preInitialize: (binding) async {
        await Future((){});
      },
      builder: (context, snapshot, _) {
       return MaterialApp();
      }
    ),
    onFlutterError: print,
  );

  final ZoneConfiguration zoneConfiguration = ZoneConfiguration(
    onZoneError: (error, stack) {
       print(error); 
       print(stack);
    },
  );

  appRunner(
    RunnerConfiguration.guarded(
      widgetConfig: widgetConfiguration,
      zoneConfig: zoneConfiguration,
    ),
  );
}

```

![image](https://github.com/MadBrains/App-Runner-Flutter/assets/34741787/a184e33d-6e64-4c15-b6ac-c15fb370dc45)

$ flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 3.10.0, on Microsoft Windows [Version 10.0.22621.1702], locale ru-RU)
[√] Windows Version (Installed version of Windows is version 10 or higher)
[√] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[√] Chrome - develop for the web
[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.5.3)
[√] Android Studio (version 2021.3)
[√] IntelliJ IDEA Ultimate Edition (version 2022.3)
[√] VS Code (version 1.78.2)
[√] Connected device (4 available)
[√] Network resources

• No issues found!

$ flutter --version
Flutter 3.10.0 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 84a1e904f4 (5 days ago) • 2023-05-09 07:41:44 -0700
Engine • revision d44b5a94c9
Tools • Dart 3.0.0 • DevTools 2.23.1

**Comment by KolasikOmetov** [comment]
Thanks for your report. I fixed this bug in 2.2.0 version. Could you try it out?

**Comment by StarProxima** [comment]
> Thanks for your report. I fixed this bug in 2.2.0 version. Could you try it out?

Sorry for delay. Everything works without errors on Flutter 3.10.2. Thanks for the fix!
